index.html

```diff
@@ -28,14 +27,31 @@ <h4>Message of the day</h4>
 </section>
 
 <div class="container my-4">
+  <div class="row">
+    <div class="col-md-12 d-flex align-items-stretch">
+      <div class="card w-100 mb-3 shadow-sm">
+        <div class="card-body">
+          <h4 class="card-title">Looking for a Hermes replacement? We've got you covered!</h4>
+          <p>The UIS have <a href="https://help.uis.cam.ac.uk/news/hermes-migration-july2020">announced the retirement</a> 
+            of the in-house Hermes email system with the preferred migration being to Microsoft's proprietary
+            Exchange Online system.</p>
+          <p>The SRCF has run its own open-source mail system, called Hades, for many years. It is fully standards 
+            (IMAP/SMTP) compliant and works with any mail client or via the web client, called Roundcube (which is 
+            what Hermes runs). Click below for more information and to forward your Cambridge email address to the 
+            SRCF.</p>
+          <a class="btn btn-secondary" role="button" href="https://docs.srcf.net/email/hades.html">Setup my @cam &raquo;</a>
+        </div>
+      </div>
+    </div>
+  </div>
+
   <div class="row">
     <div class="col-md-4 d-flex align-items-stretch">
       <div class="card mb-3 shadow-sm">
         <div class="card-body">
           <h4 class="card-title">Services</h4>
-          <p>Our volunteers maintain a selection of free services for thousands of members of the university. To name a
```

Comment:
There's a lot of gratutious autoformat in this file (and trailing spaces, which I'm not opinionated about but aren't the norm here), from this point below.  I _think_ the only real change is the Hades plug above, if I've not missed anything.  Should we avoid the churn?

Reply:
Sorry -- there is also the removal of the meetings card